test/e2e/data/infrastructure-azure/v1alpha4/cluster-template-kcp-adoption.yaml

```diff
@@ -0,0 +1,347 @@
+apiVersion: v1
```

Comment:
Where are these files coming from btw? Are they related to this PR? If not it might be something going on with `make generate`.